src/sqlfluff/rules/L018.py

```diff
@@ -22,25 +21,23 @@
 @document_fix_compatible
 @document_configuration
 class Rule_L018(BaseRule):
-    """``WITH`` clause closing bracket should be aligned with ``WITH`` keyword.
+    """``WITH`` clause closing bracket should be on a new line.
```

Comment:
As mentioned elsewhere I think the rule description should be changed to "Common Table Expressions should have the closing bracket aligned to the opening bracket line". Or something like that.

Reply:
Ah I missed that. In that case a lot of my review may be rubbish :-)